src/unix/tty.c

```diff
@@ -39,6 +39,7 @@ int uv_tty_init(uv_loop_t* loop, uv_tty_t* tty, int fd, int readable) {
   int flags;
   int newfd;
   int r;
+  char path[256] = { 0 };
```

Comment:
No, ttyname_r will have a return value of !=0 if it fails to write the name completely. So we can skip this.